data-pipeline/src/pipeline/mappings.py

```diff
@@ -263,3 +259,49 @@ def map_cost_series(category_name, df, basis):
         )
 
     return df
+
+
+_inner_las = [201, 202, 204, 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 309, 316]
+_outer_las = [
+    203,
+    301,
+    302,
+    303,
+    304,
+    305,
+    306,
+    307,
+    308,
+    310,
+    311,
+    312,
+    313,
+    314,
+    315,
+    317,
+    318,
+    319,
+    320,
+]
+
+
+def map_london_weighting(la: int, estab: int):
+    if pd.isnull(la):
+        return "Neither"
+    else:
+        match f"{la}{estab}":
+            case "3066900":
+                return "Outer"
+            case "9332064":
+                return "Neither"
+            case "8946900":
+                return "Neither"
+            case "3906900":
+                return "Neither"
+            case _:
```

Comment:
What is the reason for focusing on these specific LAEstabs in particular?

Reply:
@colinbull was there a reason for these LAEstabs?

Reply:
They are taken from the SFB script that generates the London weighting column. I think the specific inclusion / exclusion is to do with bourgh changes.